### DISCUSSION TITLE
How to make Button Message to reply?

### DISCUSSION THREAD

**Discussion by OnezIAm** [tulir/whatsmeow#557] [WhatsApp protocol Q&A]
I tried so many example from this QnA, but everytime I got the obsolete one I think, because it is not working for me,

here is the code example for now, and the bot did not reply anything to me, please help

`
func eventHandler(evt interface{}) {
	switch v := evt.(type) {
	case *events.Message:

		if v.Info.IsFromMe {
			fmt.Println("Received a message!", v.Message.GetConversation())
			msg := &waProto.Message{
				ListMessage: &waProto.ListMessage{
					Title:       proto.String("ListMessage title"),
					Description: proto.String("ListMessage Description"),
					FooterText:  proto.String("ListMessage footer"),
					ButtonText:  proto.String("ListMessage ButtonText"),
					ListType:    waProto.ListMessage_SINGLE_SELECT.Enum(),
					Sections: []*waProto.ListMessage_Section{
						{
							Title: proto.String("Section1 title"),
							Rows: []*waProto.ListMessage_Row{
								{
									RowId:       proto.String("id1"),
									Title:       proto.String("ListMessage section row title"),
									Description: proto.String("ListMessage section row desc"),
								},
								{
									RowId:       proto.String("id2"),
									Title:       proto.String("title 2"),
									Description: proto.String("desc 2"),
								},
							},
						},
						{
							Title: proto.String("Section2 title"),
							Rows: []*waProto.ListMessage_Row{
								{
									RowId:       proto.String("id1"),
									Title:       proto.String("ListMessage section row title"),
									Description: proto.String("ListMessage section row desc"),
								},
								{
									RowId:       proto.String("id2"),
									Title:       proto.String("title 2"),
									Description: proto.String("desc 2"),
								},
							},
						},
					},
				},
			}
			

			_, err := client.SendMessage(context.Background(), v.Info.Sender, msg)
			if err != nil {
				fmt.Println("Error sending message:", err)
			}
			
			client.SendMessage(context.Background(), v.Info.Sender, msg)

		}
	}
}`

**Comment by arugaz** [comment]
#534
